webook/arrangement/views/calendar_views.py

```diff
@@ -28,6 +32,56 @@ def __init__(self) -> None:
         self.section = get_section_manifest()
 
 
+
+class EventSourceViewMixin(ListView):
+    """ A mixin for views that serve the express purpose of serving events to FullCalendar calendars """
+
+    # Handle start and end constraining of the queryset in the GET method
+    # Set to False if you want to get all regardless of supplied time constraints, or do your own 
+    # handling of constraints in the get_queryset method.
```

Comment:
No issue to add a docstring on multiple lines, just follow the pep-8 line length for docstrings.

Reply:
#222